### PR TITLE
Salt manager #13389 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Merge fragments about /srv/susemanager/salt/

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>
https://github.com/SUSE/spacewalk/issues/13389

### CODE REVIEW

**Review by Bischoff — COMMENTED** [review]
It looks good to me, but could a developer specialist of Salt states please have a look? I really do not feel comfortable enough with the topic to approve.

File: .changelog [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 # Last packaged: 2020-11-27
 
+- Salt states chapter in Salt Guide for clarity (bsc#1179692)
```
Comment (Loquacity) [inline .changelog:9]:
```suggestion
- Edited Salt states chapter in Salt Guide for clarity (bsc#1179692)
```

File: modules/salt/nav-salt-guide.adoc [hunk 2]
```diff
@@ -13,7 +13,7 @@ endif::[]
 ** xref:salt-terminology.adoc[Terminology]
 ** xref:salt-command.adoc[The Salt Command]
 ** xref:salt-useful-commands.adoc[Common Salt Commands]
-** xref:salt-states.adoc[Salt States]
+** xref:salt-states.adoc[Salt States and Pilars]
```
Comment (Loquacity) [inline modules/salt/nav-salt-guide.adoc:16]:
```suggestion
** xref:salt-states.adoc[Salt States and Pillars]
```

File: modules/salt/pages/salt-states.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [[salt-states]]
-= Salt States
+= Salt States and Pilars
```
Comment (Loquacity) [inline modules/salt/pages/salt-states.adoc:2]:
```suggestion
= Salt States and Pillars
```

File: modules/salt/pages/salt-states.adoc [hunk 4]
```diff
@@ -31,25 +31,38 @@ Leading to these possible scenarios:
...
-├── manager_org_TESTING
-│   ├── files
-│   │   └── motd     # user created
-│   │    ... other files needed by states ...
-│   └── motd.sls     # user created
-            ... other SLS files ...
+spacecmd group_details
+----
+
+In these examples we will use an example Group ID of [systemitem]``GID``.
```
Comment (Loquacity) [inline modules/salt/pages/salt-states.adoc:45]:
```suggestion
These examples use an example Group ID of [systemitem]``GID``.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Comment by keichwa** [comment]
@brejoc can you please review this change--you are our Salt expert :)

File: modules/salt/pages/salt-file-locations.adoc [hunk 5]
```diff
@@ -72,13 +72,30 @@ It is the Salt equivalent of the {productname} database.
 Do not edit or add custom Salt data to this directory.
 ====
 
+Within this directory, each organization has a sub-directory.
+
+.Example: SLS File Directory Structure
+----
+├── manager_org_DEVEL
```
Comment (mcalmer) [inline modules/salt/pages/salt-file-locations.adoc:79]:
```suggestion
├── manager_org_<org id>
```

The organization ID was used, not the name.

**Review by Loquacity — APPROVED** [review]

**Pull request merged**